impl/src/main/java/org/ehcache/impl/internal/store/heap/Backend.java

```diff
@@ -56,7 +56,7 @@
    *
    * @return the number of mappings
    */
-  int size();
```

Comment:
This method should probably be renamed to something like `mappingCount()` as I find `size()` vs `naturalSize()` rather confusing.